README.md

```diff
@@ -34,15 +34,23 @@ and after a few moments everything should be up and ready. The script will creat
 Install:
 
 ``` bash
-dnf install -y podman podman-plugins podman-compose
+dnf install -y podman podman-plugins podman-compose podman-docker
+```
+
+Enable socket environment:
+
+``` bash
+systemctl --user enable --now podman.socket
+export DOCKER_HOST=unix:///run/user/${UID}/podman/podman.sock
+export TESTCONTAINERS_RYUK_DISABLED=true
```

Comment:
You can disable Ryuk in some properties file in the test JAR, so that user does not have set it here.